src/components/HOC/WithProfile/WithProfile.container.ts

```diff
@@ -33,7 +33,7 @@ const mapStateToProps = (state: RootState, ownProps: OwnProps): MapStateProps =>
 }
 
 const mapDispatch = (dispatch: MapDispatch, ownProps: OwnProps) => ({
-  onFetchProfile: () => dispatch(enhancedFetchProfileRequest(ownProps.router.params.profileAddress as string))
+  onFetchProfile: () => dispatch(enhancedFetchProfileRequest(ownProps.router.params.profileAddress?.toLowerCase() as string))
```

Comment:
Can the profileAddress ever be `undefined`?

Reply:
If so it might be best to use `!` instead of `?`. The conversion `as string` would still mess things up if the value was undefined.

Reply:
It can't, but the type that comes from the parameters is always optional. We've prevented the use of `!`, that's why I had to use the `?` operator.